mozilla/blurts-server
# MNTOR-1594: Add OptinMonster embed code

# References: 
Jira: [MNTOR-1594](https://mozilla-hub.atlassian.net/browse/MNTOR-1594)



# Description

This PR adds the OptinMonster embed code. The S&P Marketing team I would like to be able to contact people about Firefox Monitor Premium with their consent so we can grow Monitor Premium.

# Screenshot

<img width="1166" alt="mntor-1594-optinmonster-embed" src="https://user-images.githubusercontent.com/13835474/236843035-65842212-c2f2-4382-872a-63b8e48172bf.png">

# How to test

There are a couple of floating banners and popups that are being prepared. For testing purposes, there is only one banner enabled for now. Rules that determine for which pages and other display preferences can be adjusted remotely via OptinMonster.

- Set the env variables: `OPTINMONSTER_ACCOUNT` and `OPTINMONSTER_USER`
- Visit `/breaches` or `/user/breaches`
- Wait for 5 seconds
- A OptionMonster banner should show up

# Checklist (Definition of Done)
